src/compose/images.ts

```diff
@@ -465,7 +465,10 @@ export class Images extends (EventEmitter as new () => ImageEventEmitter) {
 		}
 	}
 
-	public static isSameImage(image1: Image, image2: Image): boolean {
+	public static isSameImage(
+		image1: Pick<Image, 'name'>,
+		image2: Pick<Image, 'name'>,
+	): boolean {
```

Comment:
These types are more accurate to what the function actually uses and what application-manager might pass in